profiles/nextflu-private/deploy.smk

```diff
@@ -0,0 +1,30 @@
+"""
+This part of the workflow handles automatic deployments of nextflu-private builds.
+"""
+
+rule all_private:
+    input:
+        jsons=_get_build_outputs(),
+    output:
+        json_dir=directory("auspice_renamed"),
+    params:
+        build_date=datetime.date.today().strftime("%Y-%m-%d"),
+    shell:
+        """
+        mkdir -p {output.json_dir};
+        for file in {input.jsons}
+        do
+            ln ${{file}} {output.json_dir}/"flu_seasonal_{params.build_date}_`basename ${{file}}`"
+        done
+        """
+
+rule deploy_all:
+    input:
+        json_dir=directory("auspice_renamed"),
```

Comment:
When I run this manually, I see a Snakemake warning:
```
$ nextstrain build . deploy_all --configfile profiles/nextflu-private.yaml -n --quiet
The flag 'directory' used in rule deploy_all is only valid for outputs, not inputs.
```

I don't think I've seen `directory()` used as an input before, I'd be wary of potential pitfalls. 
Is there a chance for`deploy_all` to run before `all_private` has completed? It might be safer to add a `touch()` file here to ensure rule order.

Reply:
Good catch, @joverlee521! This was a copy/paste typo on my part and is fixed in https://github.com/nextstrain/seasonal-flu/commit/d149b2de52620b8f6f8b675e0215e3cbe01e79e7. The rule order should be fine here, since the renamed directory has to be created by the renaming rule before deploy can happen.